packages/console/src/hooks/use-custom-domain.ts

```diff
@@ -51,10 +58,22 @@ const useCustomDomain = (autoSync = false) => {
   );
 
   return {
+    /**
+     * Legacy single custom domain.
+     * - Represents the first (and historically only) custom domain returned by the API.
+     * - Retained temporarily for backward compatibility while the multiple custom domains feature
+     *   is being rolled out (currently only enabled for selected enterprise customers).
+     * - Will be removed once multi-domain support is generally available.
```

Comment:
Can we use dev feature guard here?

Reply:
This feature is currently controlled by a dedicated feature flag, so the dev feature guard doesn’t apply here. We’re using the `MULTIPLE_CUSTOM_DOMAINS_ENABLED` environment variable to control this feature, and there’s already a guard in place before this field is used.